.circleci/config.yml

```diff
@@ -12,14 +12,23 @@ jobs:
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
+      - restore_cache:
+          key: "v2-gems"
       - run:
-          name: Fetch CocoaPods Specs
-          command: |
-            curl https://cocoapods-specs.circleci.com/fetch-cocoapods-repo-from-s3.sh | bash -s cf
+          name: Install with bundler
```

Comment:
I got this sample from https://circleci.com/docs/2.0/ios-migrating-from-1-2/#example-20-ios-project-configuration

Reply:
Yeah, I definitely am seeing the downsides of using the `pod outdated` method, I'll probably put this back if I can't find a better way to cache the checkout